server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -111,7 +111,7 @@ fn generate_response_requisition(
     record_for_processing: &RequisitionTransferProcessorRecord,
 ) -> Result<RequisitionRow, RepositoryError> {
     let store_id = record_for_processing.other_party_store_id.clone();
-    let name_link_id = request_requisition.store_row.name_id.clone();
+    let name_link_id = request_requisition.store_row.name_link_id.clone();
```

Comment:
Use `request_requisiton.name_row.id` and inline please:

```rust
RequisitionRow {
  ...
  name_link_id: request_requisiton.name_row.id.clone(),
  ...
```

Reply:
again... not the correct name id...